src/Jobs/Collection.php

```diff
@@ -133,6 +133,19 @@ public function tagged(string $tag, int $offset = 0, int $limit = 25): array
         return $response['jobs'];
     }
 
+    /**
+     * Reads jobs in a worker.
+     *
+     * @param string $worker
+     * @return array  BaseJob[]
+     */
+    public function fromWorker(string $worker): array
+    {
+        $jids = json_decode($this->client->worker_jobs($worker), true) ?: [];
```

Comment:
@JiSoft Could you please add annotation to the Client to avoid failing test.
```
 ------ ---------------------------------------------------------- 
  Line   Jobs/Collection.php                                       
 ------ ---------------------------------------------------------- 
  144    Call to an undefined method Qless\Client::worker_jobs().  
 ------ ---------------------------------------------------------- 
```

```php
/**
 * ...
 * @method array worker_jobs(string $worker)
 */
```

Btw, this will provide autocompletion for modern IDEs.  


How about a minimal test case covers this feature?

---

@osadchyi-s Please consider amending change log and documentation (as well as for #72) at the end.

Reply:
Fixed